lib/primer/open_project/forms/dsl/input_methods.rb

```diff
@@ -77,12 +79,18 @@ def decorate_options(include_help_text: true, help_text_options: {}, **options)
             if include_help_text && supports_help_texts?(form.model)
               attribute_name = help_text_options[:attribute_name] || options[:name]
               options[:label] = form.wrap_attribute_label_with_help_text(options[:label], attribute_name)
+              options[:caption] ||= help_text_caption_for(attribute_name)
```

Comment:
This approach won't be compatible with [Caption templates](https://primer.style/view-components/lookbook/pages/forms/caption_templates). We're not currently using them in (m)any places, but I did take advantage of this feature in https://github.com/opf/openproject/pull/21047.